elyra/pipeline/validate.py

```diff
@@ -276,7 +278,7 @@ async def _validate_node_properties(self, pipeline: dict, pipeline_runtime: str,
                                                                   response=response)
 
                         # Check label against kfp naming standards
-                        if pipeline_runtime == 'kfp' and filename != node_label:
+                        if pipeline_runtime == 'kfp' and node_label and filename != node_label:
```

Comment:
Per above, it seems like `node_label` should be a required value, especially since it's used in all the validation messaging that happens prior to this point.

Reply:
yep, this is going to be removed. thx